code/controllers/subsystems/responseteam.dm

```diff
@@ -2,123 +2,92 @@
 		return
 	
 	ert_count++
 	feedback_inc("responseteam_count")
 
-	var/ert_chance = progression_chance + config.ert_base_chance // Is incremented by fire.
-	ert_chance += config.ert_scaling_factor_dead*percentage_dead // the more people are dead, the higher the chance
-	ert_chance += config.ert_scaling_factor_antag*percentage_antagonists // the more antagonists, the higher the chance
-	ert_chance *= config.ert_scaling_factor
-	ert_chance = min(ert_chance, 100)
+	command_announcement.Announce("A distress beacon has been launched. Please remain calm, a relief team will arrive soon.", "[current_map.boss_name]", 'sound/effects/distressbeacon.ogg')
 
-	command_announcement.Announce("It would appear that an emergency response team was requested for [station_name()]. We will prepare and send one as soon as possible.", "[current_map.boss_name]")
-
-	if(forced_choice)
-		ert_type = forced_choice
-	else if(!forced_choice && !prob(ert_chance)) 
-		ert_type = "TCFL"
+	if(forced_choice && forced_choice != "Random")
+		for(var/datum/responseteam/R in available_teams)
+			if(R.name == forced_choice)
+				picked_team = R
+				break
+	else
+		picked_team = pick_random_team()
 
 	feedback_set("responseteam[ert_count]",world.time)
-	feedback_add_details("responseteam[ert_count]","BC:[config.ert_base_chance]")
-	feedback_add_details("responseteam[ert_count]","PGC:[progression_chance]")
-	feedback_add_details("responseteam[ert_count]","PGG:[pg_green]")
-	feedback_add_details("responseteam[ert_count]","PGY:[pg_yellow]")
-	feedback_add_details("responseteam[ert_count]","PGR:[pg_red]")
-	feedback_add_details("responseteam[ert_count]","PGD:[pg_delta]")
-	feedback_add_details("responseteam[ert_count]","PA:[percentage_antagonists]")
-	feedback_add_details("responseteam[ert_count]","PAF:[config.ert_scaling_factor_antag]")
-	feedback_add_details("responseteam[ert_count]","PD:[percentage_dead]")
-	feedback_add_details("responseteam[ert_count]","PDF:[config.ert_scaling_factor_dead]")
-	feedback_add_details("responseteam[ert_count]","SF:[config.ert_scaling_factor]")
-	feedback_add_details("responseteam[ert_count]","RT:[ert_type]")
 
 	can_call_ert = FALSE // Only one call per round, gentleman.
 	send_emergency_team = 1
 
-	sleep(600 * 5)
-	send_emergency_team = 0 // Can no longer join the ERT.
-	ert_type = "NT-ERT"
+	handle_spawner()
+
+	sleep(120 SECONDS)
+
+	for(var/datum/ghostspawner/G in sent_teams)
+		G.disable()
+
+/datum/controller/subsystem/responseteam/proc/handle_spawner()
+	for(var/N in typesof(picked_team.spawner)) //Find all spawners that are subtypes of the team we want.
+		var/datum/ghostspawner/human/ert/new_spawner = new N
+		for(var/role_spawner in SSghostroles.spawners)
+			if(new_spawner.short_name == role_spawner) //Create the spawner, then use its name to find the spawner in SSghostroles' spawner lists.
+				var/datum/ghostspawner/human/ert/good_spawner = SSghostroles.spawners[role_spawner]
+				sent_teams += good_spawner //Enable that spawner.
```

Comment:
Ensure to clear sent teams before using it in this proc (and after verifying that another team isnt in the process of being sent)

Reply:
Solved